config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -877,9 +877,9 @@ periodics:
     base_ref: master
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/generic_autobump:v20210308-634c5e5
+    - image: gcr.io/k8s-prow/generic-autobumper:v20210318-ffb8032f91
       command:
-      - /generic_autobump
+      - /app/prow/cmd/generic-autobumper/app.binary
```

Comment:
Nit: Thanks to these 2 lines you could just use `/generic_autobumper`
https://github.com/kubernetes/test-infra/blob/be35a198b9eb44e9f518d119ed129de19df88ef8/prow/cmd/generic-autobumper/BUILD.bazel#L45-L46